tumblr_client.gemspec

```diff
@@ -4,14 +4,14 @@ require File.join(File.dirname(__FILE__), 'lib/tumblr/version')
 Gem::Specification.new do |gem|
   gem.add_dependency 'faraday', '~> 0.9.0'
   gem.add_dependency 'faraday_middleware', '~> 0.9'
-  gem.add_dependency 'json'
-  gem.add_dependency 'simple_oauth'
-  gem.add_dependency 'oauth'
-  gem.add_dependency 'mime-types'
-  gem.add_development_dependency 'rake'
-  gem.add_development_dependency 'rspec'
-  gem.add_development_dependency 'webmock'
-  gem.add_development_dependency 'simplecov'
+  gem.add_dependency 'json', '~> 1.8'
```

Comment:
This may block upgrading of ruby to 2.4, which will come with json 2.0.x and won't work with json <2.0 due to Integer class unification.